docs/installation-and-operations/changing-to-bim-edition/README.md

```diff
@@ -77,3 +77,33 @@ OpenProject installations with a valid Enterprise Edition token can switch to th
 theme.
 
 Go to _-> Administration -> Design_ and from the _Themes_ drop down menu choose _OpenProject BIM_.
+
+### Docker
+
+##### OpenProject BIM Edition with Docker Compose
+
+If you would like to use OpenProject BIM Edition inside your docker-compose containers, in your `docker-compose.yml` file in `x-op-app` > `environment` add the line
```

Comment:
As asked below, wouldn't it be a good idea to use a `docker-compose.override.yml` file for that sort of configuration/adoptions?

Reply:
@machisuji please look into this